regression/.coveragerc

```diff
@@ -0,0 +1,28 @@
+[run]
+source = SUAVE
+
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Have to re-enable the standard pragma
+    pragma: no cover
+
+    # Don't complain about missing debug-only code:
+    def __repr__
+    if self\.debug
+
+    # Don't complain if tests don't hit defensive assertion code:
+    raise AssertionError
+    raise NotImplementedError
+
+    # Don't complain if non-runnable code isn't run:
```

Comment:
I like this file.  However does this code skip code that may need to be regressed?

Reply:
I'm actually not certain about what this code does. These exclude lines are part of the starter code.